modules/multi-arch-creating-podplacmement-config-using-cli.adoc

```diff
@@ -0,0 +1,82 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multiarch-tuning-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-creating-podplacement-config-using-cli_{context}"]
+
+= Creating a ClusterPodPlacementConfig CR by using the CLI
+
+You can create a `ClusterPodPlacementConfig` custom resource (CR) by using the {oc-first}.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have installed the {oc-first}.
+* You have installed the Multiarch Tuning Operator.
+
+.Procedure
+
+. Log in to the {oc-first}.
+
+. Create a `ClusterPodPlacementConfig` CR YAML file:
++
+.Example `ClusterPodPlacementConfig` CR YAML file
+[source,yaml]
+----
+apiVersion: multiarch.openshift.io/v1alpha1
+kind: ClusterPodPlacementConfig
+metadata:
+  name: cluster
+spec:
+  logVerbosityLevel: Normal <1>
+  namespaceSelector: <2>
+    matchExpressions:
+      - key: multiarch.openshift.io/exclude-pod-placement 
+        operator: DoesNotExist 
+----
+<1> Optional: You can set it to `Normal`, `Debug`, `Trace`, or `TraceAll`. It is set to `Normal` by default. 
+<2> Optional: You can configure the `namespaceSelector` to select the namespaces in which the Multiarch Tuning Operator's pod placement operand must process the `nodeAffinity` of the pods. All namespaces are considered by default.
++
+In this example, if the `multiarch.openshift.io/exclude-pod-placement` label is set in a namespace, the operand does not process the `nodeAffinity` of the pods in that namespace. Instead, the operand processes the `nodeAffinity` of the pods in namespaces that do not contain the label.
++
+If you want the operand to process the `nodeAffinity` of the pods only in specific namespaces, you can configure the `namespaceSelector` as follows:
++
+[source,yaml]
+----
+namespaceSelector:
+  matchExpressions:
+    - key: multiarch.openshift.io/include-pod-placement
+      operator: Exists  
+----
++
+In this example, the operand processes the `nodeAffinity` of the pods only in namespaces that contain the `multiarch.openshift.io/include-pod-placement` label. 
++
+[IMPORTANT]
+====
+The namespaces starting with `openshift-`, `kube-`, and `hypershift-` are excluded.
+====
+
+. Create the `ClusterPodPlacementConfig` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <file_name> <1>
+----
+<1> Replace `<file_name>` with the name of the `ClusterPodPlacementConfig` CR YAML file.
+
+.Verification
+
+* To check that the `ClusterPodPlacementConfig` CR is created, run the following command:
++
+[source,terminal]
+----
+$ oc get podplacementconfig -n <namespace>
```

Comment:
```suggestion
$ oc get clusterpodplacementconfig -n <namespace>
```

Reply:
try to ensure any other instances are not left.

Reply:
Done!